QSB/TransformSync/TransformSync.cs

```diff
@@ -82,6 +82,11 @@ private void Update()
             // Get which sector should be used as a reference point
             var sectorTransform = PlayerRegistry.GetSector(PlayerTransformSync.LocalInstance.netId.Value);
 
+            if (sectorTransform == null)
+            {
+                DebugLog.ToConsole($"Error - Player ID {PlayerTransformSync.LocalInstance.netId.Value}'s reference sector is null!", OWML.Common.MessageType.Error);
```

Comment:
`using OWML.Common;`, then shorten `OWML.Common.MessageType.Error` to `MessageType.Error`